src/common/reducers/auth-store.js

```diff
@@ -3,6 +3,7 @@ import * as ActionTypes from '../actions/auth-store';
 export function authStore(state = {
   isFetching: false,
   hasDischarge: false,
+  signingOut: false,
```

Comment:
Do we need that information?
It doesn't seem that we use it anywhere, so I don't think it's necessary it add.